.idea/watcherTasks.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
No, and with goland I tend not to anymore, primarily because it's possible to expose the actual build / execution configurations to git. So if you have typical build / execution scenarios, (I literally have about 55 for Babylon. Not included thankfully), you can make sure they're included here. 

Plus for other IDEs like IntelliJ you literally can't operate without them. IntelliJ stores the project definitions for Java projects (module definitions, dependencies, build artifact structures, etc) in .idea. So I'm kind of used to keeping them with the actual code.